0000-hashed-timelock-agreements.md

```diff
@@ -0,0 +1,116 @@
+# Hashed-Timelock Agreements (HTLAs)
+
+> Generalization of [Hashed-Timelock Contracts (HTLCs)](https://en.bitcoin.it/wiki/Hashed_Timelock_Contracts) used to secure Interledger payments.
+
+Interledger uses conditional transfers to secure multi-hop payments. This document describes how Interledger payments can be implemented over any type of ledger, including those that do not provide support for conditional transfers. Hashlocks and timeouts can be enforced by ledgers or they can be upheld by senders and recipients themselves.
+
+Before reading this document, it is important to understand how Interledger isolates participants from risks posed by indirect connections. The tradeoffs listed here are only the concern of parties that are _directly connected_. For more on the security model, see the [Interledger Architecture](./0001-interledger-architecture/0001-interledger-architecture.md).
```

Comment:
Would it be worth including a section in this doc about how HTLAs enable secure multi-hop payments? Or is it sufficient to point to other Interledger docs?